apps/editing-toolkit/editing-toolkit-plugin/block-patterns/src/premium-block-patterns.tsx

```diff
@@ -40,12 +45,16 @@ export const PatternTitleContainer: React.FunctionComponent< PatternTitleProps >
 export const PremiumBlockPatterns: React.FunctionComponent = () => {
 	const { __experimentalBlockPatterns } = useSelect( ( select ) =>
 		select( 'core/block-editor' ).getSettings()
+	) as ExperimentalEditorSettings;
+	const {
```

Comment:
I have to make this change so we can use `__experimentalBlockPatterns`. I don't know if there is a better approach